Classes/WPEditorView.m

```diff
@@ -217,7 +219,15 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
                 });
             }
         }
-    }
+	}
+}
+
+- (void)startObservingTitleFieldChanges
+{
+	[[NSNotificationCenter defaultCenter] addObserver:self
+											 selector:@selector(titleTextDidChange)
+												 name:UITextFieldTextDidChangeNotification
+											   object:nil];
```

Comment:
I think the latest versions of Xcode are ignoring the project default for spaces vs tabs.